CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinViewModel.swift

```diff
@@ -15,6 +15,7 @@ import UseCaseProtocol
 
     @Published public private(set) var coinDetail: CoinDetailItem?
     @Published public private(set) var chartData: [ChartDataPointUIModel] = []
+    @Published public private(set) var isSuccess = false
```

Comment:
Please make the flag name more specific for better clarification 🙏
```suggestion
    @Published public private(set) var isCoinDataRefreshedSuccessfully = false
```

Reply:
>And in case the refresh is unsuccessful, we might need to keep showing the same data and keep the same filter also.

As we handle the refresh in unsuccessful requests, I think we should handle it too when we switch among timeframes in the following video:
https://user-images.githubusercontent.com/25881847/212600288-d60ec3ee-1af8-4f81-9e0e-67d0749895c4.mp4
What do you think? 🤔

Reply:
> I think we should handle it too when we switch among timeframes in the following video

That is definitely a good point @nkhanh44 👍 Now the need for having a popup message arises critically as the user might get confused why they can't switch the filter, but yeah I think we can uniformly handle the error popups in a separated PR 😁